testdata/config/quota.yaml

```diff
@@ -17,7 +23,7 @@ metadata:
 spec:
   dimensions:
     source: source.labels["app"] | "unknown"
```

Comment:
with the raw vm work, i'm wondering if we should switch from labels to `source.service` (at least for 0.2).

Reply:
how about `source.labels["app"] | source.service`

Reply:
how about `source.labels["app"] | source.service | "unknown"` to cover all bases?

Reply:
done.